.circleci/config.templ.yml

```diff
@@ -788,7 +788,7 @@ jobs:
 
   elasticsearch:
     <<: *machine_executor
-    parallelism: 4
+    parallelism: 17
```

Comment:
How do you choose this number?  We make it around 4 times faster to align with how fast the other jobs run?

Reply:
A number larger than 10 (each job has fewer than 10 hashes) that evenly divides 102 (why not)